api/src/routing.js

```diff
@@ -111,6 +112,19 @@ app.postJsonOrCsv = (path, callback) => handleJsonOrCsvRequest('post', path, cal
 app.postJson = (path, callback) => handleJsonRequest('post', path, callback);
 app.putJson = (path, callback) => handleJsonRequest('put', path, callback);
 
+app.use(prometheusMiddleware({
+  metricsPath: '/api/v1/prometheus_metrics',
```

Comment:
Can we change the name to be more specific? The monitoring endpoint is already consumed by prometheus so it'd be good to be clear on the distinction. Also other endpoints are using hyphens not underscores. 

Is it possible to combine the two together or does this middleware require its own endpoint? If it must be distinct, then let's use something like `/api/v1/express-metrics` or `/api/v1/endpoint-performance`?

Reply:
I like express-metrics. Will update